.github/workflows/artifacts.yml

```diff
@@ -0,0 +1,77 @@
+name: Build artifacts
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  alpine_build:
+    name: Alpine Build
+    runs-on: ubuntu-16.04
+    steps:
+    - uses: actions/checkout@v2
+    - run: docker-compose run build
+    - run: docker-compose run Profiler.Alpine
+    - run: docker-compose run package.alpine
+    - run: mkdir artifacts
+    - run: cp -a deploy/linux/. artifacts/
+    - uses: actions/upload-artifact@v2
+      with:
+        name: artifacts
+        path: artifacts
+
+  linux_build:
+    name: Linux Build
+    runs-on: ubuntu-16.04
+    steps:
+    - uses: actions/checkout@v2
+    - run: docker-compose run build
+    - run: docker-compose run Profiler
+    - run: docker-compose run package
+    - run: mkdir artifacts
+    - run: cp -a deploy/linux/. artifacts/
+    - uses: actions/upload-artifact@v2
+      with:
+        name: artifacts
+        path: artifacts
+
+  windows_build:
+    name: Windows Build
+    runs-on: windows-2019
+    steps:
+      - uses: actions/checkout@v2
+      - uses: microsoft/setup-msbuild@v1.0.2
+      - run: dotnet --list-sdks
+        shell: cmd
+      - run: choco install wixtoolset
+        shell: cmd
+      - run: nuget restore Datadog.Trace.sln
+        shell: cmd
+      - run: nuget restore src\OpenTelemetry.DynamicActivityBinding\OpenTelemetry.DynamicActivityBinding.sln
+        shell: cmd
+      - run: msbuild Datadog.Trace.proj /t:BuildCsharp /p:Configuration=Release
+        shell: cmd
+      - run: dotnet pack -c Release -o src\bin\SignalFx.Tracing src\Datadog.Trace\Datadog.Trace.csproj
+        shell: cmd
+      - run: dotnet pack -c Release -o src\bin\SignalFx.Tracing.OpenTracing src\Datadog.Trace.OpenTracing\Datadog.Trace.OpenTracing.csproj
+        shell: cmd
+      - run: msbuild Datadog.Trace.proj /t:BuildCpp /p:Configuration=Release;Platform=x64
+        shell: cmd
+      - run: msbuild Datadog.Trace.proj /t:BuildCpp /p:Configuration=Release;Platform=x86
+        shell: cmd
+      - run: msbuild Datadog.Trace.proj /t:msi /p:Configuration=Release;Platform=x64
+        shell: cmd
+      - run: msbuild Datadog.Trace.proj /t:msi /p:Configuration=Release;Platform=x86
+        shell: cmd
+      - run: msbuild Datadog.Trace.proj /t:CreateHomeDirectory /p:Configuration=Release;Platform=x64
+        shell: cmd
+      - run: mkdir artifacts
+        shell: cmd
+      - run: cp -r src/WindowsInstaller/bin/Release/x64/en-us/* artifacts
+        shell: cmd
+      - run: cp -r src/WindowsInstaller/bin/Release/x86/en-us/* artifacts
+        shell: cmd
```

Comment:
can you extract it to some script (even batch file) so that we can also use it locally?

Reply:
A cmd file will be my preference, it is simple and the commands above are pretty straightforward, be sure to separate the setup steps, e.g. choco install. Suggestion make Configuration a parameter (either as env var or parameter to the cmd with the default being Debug, for us :) to use while developing)

Reply:
Will somebody with windows test it locally when I done? :)

Reply:
YES!!!! Just put here in the PR and I will give it a go in ... 8-9 hours? 💤